.github/workflows/create-release.yml

```diff
@@ -39,3 +46,16 @@ jobs:
               ref: 'refs/tags/${{ env.current-version }}',
               sha: '${{ github.sha }}'
             })
+
+      - name: Create release
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.generate-token.outputs.token }}
+          script: |
+            github.rest.repos.createRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag_name: '${{ env.current-version }}',
+              draft: false,
+              prerelease: false
```

Comment:
A possible enhancement here would be to add a `.github/release.yml` configuration file and enable [automatic release note generation](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) with `generate_release_notes: true`